docs/manual.md

```diff
@@ -299,8 +323,13 @@ $ apalache check --inv=Safety \
   --length=20 --cinit=ConstInit y2k_cinit.tla
 ```
 
-**Parameterized initialization**. As a bonus of this approach, Apalache allows
-one to check a specification over a bounded set of parameters.  For example:
+### Parameterized initialization
+
+<!-- TODO(shonfeder) How does this subsection relate to the example? how would it -->
```

Comment:
@konnov or @andrey-kuprianov for my own edification, I'm hoping to get a bit of clarification on this point.

Reply:
It's that you can initialize not with a single value, but with a constraint on initial values. Compared to TLC, this gives you more flexibility

Reply:
I understand now, enough to remove my question but not quite enough to add any additional clarification. I'll revisit this bit once I have more experience. Thanks :)